include/tscore/Filenames.h

```diff
@@ -25,6 +25,19 @@
 // Configuration file names
 extern const char *STORAGE_CONF_FILENAME;
 extern const char *RECORDS_CONF_FILENAME;
+extern const char *VOLUME_CONF_FILENAME;
+extern const char *PLUGIN_CONF_FILENAME;
+
+extern const char *LOGGING_CONF_FILENAME;
+extern const char *CACHE_CONF_FILENAME;
+extern const char *IPALLOW_CONF_FILENAME;
+extern const char *HOSTING_CONF_FILENAME;
+extern const char *SOCKS_CONF_FILENAME;
+extern const char *PARENT_CONF_FILENAME;
+extern const char *REMAP_CONF_FILENAME;
+extern const char *MULTICERT_CONF_FILENAME;
+extern const char *SPLITDNS_CONF_FILENAME;
+extern const char *SNI_CONF_FILENAME;
```

Comment:
I think code is more readable when include files avoid containing information that doesn't strictly need to be in them.  But @SolidWallOfCode will name a pony on his ranch after you if you use constexpr instead of const.

Reply:
If you're going to do that, what would clearly better is
```
constexpr std::string_view STORAGE_CONF_FILENAME = "storage.config"
// ...
```

Reply:
Then how do you pass a NULL terminated string to functions that require it?

Reply:
I guess it is NULL terminated by default, but since you can't guarantee a std::string_vew is NULL terminated it seems ugly to me.

Reply:
Fwiw, I did try putting the strings in the .h file (albeit, not with the constexpr, maybe that's the magic). But, when I tried that, you get link errors, because of duplicated symbols (the include file is included in multiple compile / link units).

I'm -1 on "auto" here though, but if constexpr char* STORAGE_CONF_FILENAME = " ..."  works in the include file without link time errors, I'm ok with that. string_view seems like a huge meh, why bother, there's zero value of string_view here.

Reply:
Well, we _could_ add a ts::TextViewZ (publicly derived from TextView) to document and do some static checking that the string should be null terminated.

Reply:
That seems silly for a static string that's perfectly char * ... What's with the obsession of making these strings text_view ? Until we have a use case for why they should be text_view (e.g. you need the length), I think we should stick to the simplest data type.

Reply:
Yes, this in particular case you'd be depending on that fact the `string_view` was initialized to point to a string that is null terminated. Nothing forbids a `string_view` from being null terminated. But whatever - I like capturing any data that might be useful at compile time, but it's not really a big deal.

Yes, `constexpr` is magic in terms of preventing multiple symbol definitions.